wallet/cardano-wallet.js

```diff
@@ -178,19 +181,27 @@ const CardanoWallet = async (options) => {
     }
   }
 
-  function estimateTxSize(txInputs, coins, outputcount) {
+  function estimateTxSize(txInputs, coins, hasChange, outAddressSize) {
```

Comment:
I would flip the order of hasChange and outAddress, since I see this order as more intuitive.